tests/models/nn/test_sasrec.py

```diff
@@ -660,14 +734,36 @@ def test_tranform_dataset_i2i(
         ),
     )
     def test_get_dataloader_train(
-        self, dataset: Dataset, data_preparator: SASRecDataPreparator, train_batch: List
+        self, dataset: Dataset, data_preparator: SASRecDataPreparator, train_batch: tp.List
     ) -> None:
-        dataset = data_preparator.process_dataset_train(dataset)
-        dataloader = data_preparator.get_dataloader_train(dataset)
+        data_preparator.process_dataset_train(dataset)
+        dataloader = data_preparator.get_dataloader_train()
         actual = next(iter(dataloader))
         for key, value in actual.items():
             assert torch.equal(value, train_batch[key])
 
+    @pytest.mark.parametrize(
+        "val_batch",
+        (
+            (
+                {
+                    "x": torch.tensor([[0, 1, 3]]),
+                    "y": torch.tensor([[2]]),
+                    "yw": torch.tensor([[1.0]]),
+                }
+            ),
+        ),
+    )
+    def test_get_dataloader_val(
+        self, dataset: Dataset, data_preparator_val_mask: SASRecDataPreparator, val_batch: tp.List
+    ) -> None:
+        data_preparator_val_mask.process_dataset_train(dataset)
+        dataloader = data_preparator_val_mask.get_dataloader_val()
+        if dataloader is not None:
```

Comment:
This is bad. You might have `get_dataloader_val` broken and you will not know it from this test. The test will pass.
Please remove `if`

Reply:
Removed `if` and added `type: ignore`, because `data_preparator_val_mask.get_dataloader_val()` can return None object, but this case isn't related to this test.